src/CVV/CVV.css

```diff
@@ -1,12 +1,21 @@
-#Modals {
-  background-color: whitesmoke;
-  height: 100vh;
+article {
```

Comment:
This needs to be more specific. This will definitely affect more things than you want it to (such as things on other pages).